.gitignore

```diff
@@ -0,0 +1,13 @@
+/.bundle/
+/.yardoc
+/_yardoc/
+/coverage/
+/doc/
+/pkg/
+/spec/reports/
+/tmp/
+
+# rspec failure tracking
+.rspec_status
+
+/Gemfile.lock
```

Comment:
Should we check this in to avoid installing different versions in AWS and locally?

Reply:
So this is the root-level Gemfile.lock (I think?), each individual function should be able to have its own Gemfile.lock

Reply:
Ah, right 👍🏼